### DISCUSSION TITLE
How can I execute a select that is not against a table

### DISCUSSION THREAD

**Discussion by jayadevanm** [PostgREST/postgrest#2000] [Q&A] [answered]
To test that PostgREST and PostgreSQL are up and running, I would like to execute a `SELECT 1` or something similar. How can this be done? I can write  a function that returns 1 and call it using rpc. But is it possible to just execute a SELECT?  Or is there a clean/quick way to do this?

**Comment by steve-chavez** [comment, marked as answer]
Check this previous discussion: https://github.com/PostgREST/postgrest/discussions/1891

https://github.com/PostgREST/postgrest/issues/1933 will address this use case.

**Reply by wolfgangwalther** [reply]
> There's now proper health check endpoints on the latest pre-release: https://github.com/PostgREST/postgrest/releases/tag/v9.0.0.20220107
